packages/oui-checkbox/src/checkbox.html

```diff
@@ -8,6 +8,7 @@
          ng-disabled="$ctrl.disabled">
   <label for="{{:: $ctrl.id }}" class="oui-checkbox__label-container">
     <span class="oui-checkbox__label" ng-bind="$ctrl.text"></span>
+    <span class="oui-checkbox__description" ng-bind="$ctrl.description" ng-if="$ctrl.description"></span>
```

Comment:
The label must be explicit and concise, 'cause the label is read when the linked input is focus.

So you will hear : "Label Description BLABLABLABLABLABLABLA - Text field - Required"

You should consider aria-describedby and keep a simple label :)

Reply:
Maybe with a structure like:
```
<div>
  <input>
  <label></label>
  <span>></label>
</div>
```

Reply:
Forget my upper comment,
https://developer.mozilla.org/fr/docs/Accessibilit%C3%A9/ARIA/Techniques_ARIA/Utiliser_l_attribut_aria-describedby

Like said @jleveugle `aria-labelledby` and `aria-describedby` seems to resolve this issue.

Reply:
Fixed, I used `aria-labelledby` and `aria-describedby` since we need the description to be clickable.